test_threadleak.py

```diff
@@ -110,23 +119,28 @@ def test_leak_disabled_marker(testdir):
         function_marker="@pytest.mark.threadleak(enabled=False)"))
     result = testdir.runpytest('-v')
     result.stdout.fnmatch_lines(['*::test_leak PASSED*'])
-    assert result.ret == 0
+    assert result.ret == ExitCode.OK
 
 
 def test_leak_enabled_module_marker(testdir):
     testdir.makepyfile(make_source(
         module_marker="pytestmark = pytest.mark.threadleak"))
     result = testdir.runpytest('-v')
     result.stdout.fnmatch_lines(['*::test_leak FAILED*'])
-    assert result.ret == 1
+    assert result.ret == ExitCode.TESTS_FAILED
 
 
 def test_leak_enabled_class_marker(testdir):
     testdir.makepyfile(make_source(
         class_marker="@pytest.mark.threadleak"))
     result = testdir.runpytest('-v')
-    result.stdout.fnmatch_lines(['*::test_leak FAILED*'])
-    assert result.ret == 1
+    print(f"""--]]]]]]]]]]]]------------
+    {result.stdout.str()}
+    ----++++++++++++++++------------
+    """)
```

Comment:
I see that you removed this debug print in a later commit, but this makes review harder and requires squashing the commits to keep clean history. It would be useful if you can squash your commits before the next review to separate logical changes.